vpr/src/route/sink_sampling.h

```diff
@@ -149,16 +149,23 @@ inline void convex_hull_downsample(ParentNetId net_id, const t_bb& net_bb, vtr::
         RRNodeId rr_sink = route_ctx.net_rr_terminals[net_id][i];
         if (!inside_bb(rr_sink, net_bb))
             continue;
-        SinkPoint point{rr_graph.node_xlow(rr_sink), rr_graph.node_ylow(rr_sink), int(i)};
+        Direction dir = rr_graph.node_direction(rr_sink);
```

Comment:
Sinks don't have direction ... not sure what this doing.  I think sinks are all 1x1 anyway -- can probably just change to (xlow,ylow), but check to be sure.